frontend/test/ui/pages/desktop/home.py

```diff
@@ -0,0 +1,21 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Home(Base):
+
+    @property
+    def header(self):
+        return self.Header(self)
+
+    class Header(Region):
+
+        _root_locator = (By.CLASS_NAME, 'banner')
+
+        _copter_locator = (By.CLASS_NAME, 'copter')
+
+        @property
+        def copter(self):
+            return self.find_element(*self._copter_locator)
```

Comment:
We shouldn't return web elements from page objects. Instead we should provide a property such as `is_copter_displayed`. This way the test only has to be concerned with the API of the page object, and not that of Selenium, etc.